platform/darwin/src/MGLAnnotation.h

```diff
@@ -36,6 +41,14 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, readonly, copy, nullable) NSString *subtitle;
 
+#if TARGET_OS_IPHONE
+
+@property (nonatomic) BOOL canShowCallout;
```

Comment:
Where these properties live will need to be reevaluated after https://github.com/mapbox/mapbox-gl-native/pull/4801, aye. Though, if callouts only existed on `MGLAnnotationView`, wouldn’t GL (e.g., `MGLPointAnnotation`) annotations not have (custom) callouts?